projects/plugins/social/src/js/editor.js

```diff
@@ -59,6 +71,24 @@ const JetpackSocialSidebar = () => {
 			} }
 		/>
 	);
+
+	// Handle when the review request is dismissed
+	const handleReviewDismiss = useCallback( () => {
+		// Save that the user has dismissed this by calling to the social plugin API method
+		apiFetch( {
+			path: reviewRequestDismissUpdatePath,
+			method: 'POST',
+			data: { dismissed: true },
+		} )
+			.then( () => {
+				// there's nothing to do here.
+			} )
+			.catch( () => {
+				// there's nothing to do here.
+			} );
+		hideReviewRequest();
```

Comment:
I don't understand why we need to have two functions `hideReviewRequest()` and `handleReviewDismiss()` that just invokes the first one. Cannot we just replace this line with a suggestion and get rid of `hideReviewRequest`?
```suggestion
		setIsReviewRequestVisible( false )
```

Reply:
Indeed. Good suggestion. I think the two functions was an artifact of adding the hide and API calls in separate steps. I missed grouping those together 👍